streams/src/test/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImplTest.java

```diff
@@ -56,28 +58,35 @@
     private static final String TOPIC = "topic";
     private final KStreamBuilder builder = new KStreamBuilder();
     private KGroupedStream<String, String> groupedStream;
+    private KStreamTestDriver driver = null;
 
     @Before
     public void before() {
         final KStream<String, String> stream = builder.stream(Serdes.String(), Serdes.String(), TOPIC);
         groupedStream = stream.groupByKey(Serdes.String(), Serdes.String());
     }
 
+    @After
+    public void cleanup() {
```

Comment:
I agree, and personally I also prefer the `ExternalResource` way and that is as I said in KAFKA-3623. But we can still debate over Ewen's suggestion v.s. `ExternalResource` since not all tests creates an instance of `driver` which makes it a bit tricky.